c_src/fast_yaml.c

```diff
@@ -151,6 +195,24 @@ static ERL_NIF_TERM zip(ErlNifEnv* env, ERL_NIF_TERM list)
 	return list;
 }
 
+static ERL_NIF_TERM map(ErlNifEnv* env, ERL_NIF_TERM pairs)
+{
+    ERL_NIF_TERM ret;
+    ERL_NIF_TERM key;
+    ERL_NIF_TERM val;
+
+     ret = enif_make_new_map(env);
```

Comment:
Indentation

Reply:
Re-indentened the file and converted any tabs to 4 spaces